spec/support/helpers.rb

```diff
@@ -481,6 +481,14 @@ def simulate_bundler_version(version)
       ENV["BUNDLER_SPEC_VERSION"] = old if block_given?
     end
 
+    def simulate_rubygems_version(version)
+      old = ENV["BUNDLER_SPEC_RUBYGEMS_VERSION"]
+      ENV["BUNDLER_SPEC_RUBYGEMS_VERSION"] = version.to_s
+      yield if block_given?
+    ensure
+      ENV["BUNDLER_SPEC_RUBYGEMS_VERSION"] = old if block_given?
```

Comment:
If no block is given, will this still return the ENV variable to its previous version?